cedar-drt/fuzz/src/schemas.rs

```diff
@@ -0,0 +1,98 @@
+use cedar_policy_validator::{ActionType, ApplySpec, NamespaceDefinition, SchemaFragment};
+
+/// Check if two schema fragments are equivalent, modulo empty apply specs.
+/// We do this because there are schemas that are representable in the JSON that are not
+/// representable in the human-readable syntax. All of these non-representable schemas
+/// are equivalent to one that is representable.
+///
+/// Example:
+/// You can have a JSON schema with an action that has no applicable principals and some applicable
+/// resources.
+/// In the human-readable syntax, you can't. The only way to write an action with no applicable
+/// principals is:
+/// ```
```

Comment:
So this doesn't get run as a doc-test.
```suggestion
/// ```cedarschema
```